### PR TITLE
Update Data Warehouse to 9.6

### PR DESCRIPTION
**Pull request by tanderegg**

### CODE REVIEW

**Comment by tanderegg** [comment]
@rioh @stirlingv Could you take a look when you have a chance?  Thanks!

File: deploy/roles/postgresql-server/tasks/main.yml [hunk 1]
```diff
@@ -1,65 +1,94 @@
...
-  when: install_ppas
+- name: Ensure the install directory exists
+  file:
+    path: "{{ pp_prefix }}/{{ pp_install_path }}"
+    state: directory
+    owner: "{{ pp_serviceaccount }}"
+    group: "{{ pp_serviceaccount }}"
+    mode: 0700
+  when: pp_install or pp_upgrade
+    tags:
```
Comment (stirlingv) [inline deploy/roles/postgresql-server/tasks/main.yml]: remove space before tags:

**Review by stirlingv — COMMENTED** [review]
Looks good, small comment about a single typo - Travis helped catch!

**Review by rioh — CHANGES_REQUESTED** [review]
Seems to be a typo on line 78

**Comment by tanderegg** [comment]
@rioh Woops, thanks!

**Pull request merged**